src/common/export/download-svg.business.ts

```diff
@@ -10,12 +10,19 @@ const initiateDownload = (url: string, name: string) => {
   document.body.removeChild(link);
 };
 
-export const downloadSvg = (svg: JSX.Element) => {
-  const svgString = ReactDOMServer.renderToStaticMarkup(svg);
-  const blob = new Blob([svgString], { type: 'image/svg+xml' });
+export const downloadFile = (
+  filename: string,
+  content: string,
+  contentType: string
+) => {
+  const blob = new Blob([content], { type: contentType });
   const svgUrl = URL.createObjectURL(blob);
+  initiateDownload(svgUrl, filename);
+};
```

Comment:
Extract this function (downloadFile) to another file, this is more generic than svg

something like

_./src/common/export/classic-download.ts_